apps/src/templates/projects/generateFakeProjects.js

```diff
@@ -41,75 +41,123 @@ export const stubFakePersonalProjectData = [
-    publishedAt: '2015-02-04T23:59:59.999-08:00',
-    featuredAt: '2016-12-31T23:59:59.999-08:00',
-    unfeaturedAt: 'null',
+    publishedAt: '2023-02-14T02:14:51.000+00:00',
+    featuredAt: '2024-02-14T02:14:51.000+00:00',
+    unfeaturedAt: null,
   },
   {
-    projectName: 'Featured Project 4',
+    projectName: 'Active Featured Project 4',
     type: 'playlab',
+    status: 'active',
     channel: 'abcd8',
-    publishedAt: '2015-02-04T23:59:59.999-08:00',
-    featuredAt: '2016-12-31T23:59:59.999-08:00',
-    unfeaturedAt: 'null',
+    publishedAt: '2022-12-16T02:14:51.000+00:00',
+    featuredAt: '2023-12-16T02:14:51.000+00:00',
+    unfeaturedAt: null,
   },
 ];
 
-export const stubFakeUnfeaturedProjectData = [
+export const stubFakeArchivedFeaturedProjectData = [
   {
-    projectName: 'Unfeatured Project 1',
+    projectName: 'Archived Featured Project 1',
     type: 'gamelab',
+    status: 'archived',
     channel: 'abcd1',
-    publishedAt: '2015-02-04T23:59:59.999-08:00',
-    featuredAt: '2016-12-31T23:59:59.999-08:00',
-    unfeaturedAt: '2017-12-31T23:59:59.999-08:00',
+    publishedAt: '2020-02-16T02:14:51.000+00:00',
+    featuredAt: '2023-02-16T02:14:51.000+00:00',
+    unfeaturedAt: '2024-02-16T02:14:51.000+00:00',
   },
   {
-    projectName: 'Unfeatured Project 2',
+    projectName: 'Archived Featured Project 2',
     type: 'applab',
+    status: 'archived',
     channel: 'abcd2',
-    publishedAt: '2015-02-04T23:59:59.999-08:00',
-    featuredAt: '2016-12-31T23:59:59.999-08:00',
-    unfeaturedAt: '2017-12-31T23:59:59.999-08:00',
+    publishedAt: '2020-02-16T02:14:51.000+00:00',
+    featuredAt: '2023-01-16T02:14:51.000+00:00',
+    unfeaturedAt: '2024-02-16T02:14:51.000+00:00',
   },
   {
     projectName:
-      "Unfeatured Project 3 has a super extra long name let's see how it looks in the table",
+      "Archived Featured Project 3 has a super extra long name let's see how it looks in the table",
     type: 'artist',
+    status: 'archived',
     channel: 'abcd3',
-    publishedAt: '2015-02-04T23:59:59.999-08:00',
-    featuredAt: '2016-12-31T23:59:59.999-08:00',
-    unfeaturedAt: '2017-12-31T23:59:59.999-08:00',
+    publishedAt: '2020-02-16T02:14:51.000+00:00',
+    featuredAt: '2022-02-16T02:14:51.000+00:00',
+    unfeaturedAt: '2024-02-16T02:14:51.000+00:00',
   },
   {
-    projectName: 'Unfeatured Project 4',
+    projectName: 'Archived Featured Project 4',
     type: 'playlab',
+    status: 'archived',
     channel: 'abcd4',
-    publishedAt: '2015-02-04T23:59:59.999-08:00',
-    featuredAt: '2016-12-31T23:59:59.999-08:00',
-    unfeaturedAt: '2017-12-31T23:59:59.999-08:00',
+    publishedAt: '2019-02-16T02:14:51.000+00:00',
+    featuredAt: '2022-02-12T02:14:51.000+00:00',
+    unfeaturedAt: '2024-02-16T02:14:51.000+00:00',
+  },
+];
+
+export const stubFakeBookmarkedFeaturedProjectData = [
+  {
+    projectName: 'Bookmarked Featured Project 1',
+    type: 'gamelab',
+    status: 'bookmarked',
```

Comment:
you could use your projectStatuses constant in these tests too to prevent having to update the strings if the status options are modified